metrics/tarantool/network.lua

```diff
@@ -60,6 +60,84 @@ local function update_network_metrics()
             utils.set_gauge('net_requests_in_stream_queue_current',
             'count of requests currently waiting in queues of streams', box_stat_net.REQUESTS_IN_STREAM_QUEUE.current)
     end
+
+    if box.stat.net.thread ~= nil then
+        local box_stat_net_per_thread = box.stat.net.thread()
+
+        for index, per_thread_mertics in pairs(box_stat_net_per_thread) do
+            collectors_list.net_sent_total =
+                utils.set_counter('net_sent_total', 'Totally sent in bytes',
+                    per_thread_mertics.SENT.total, {thread = index})
```

Comment:
I think it's better to use explicit `tostring` casts to a metric label since only string labels are supported with most exporters.